cmd/create/cluster/cmd.go

```diff
@@ -1988,7 +2028,9 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string) string {
 	if len(spec.SubnetIds) > 0 {
 		command += fmt.Sprintf(" --subnet-ids %s", strings.Join(spec.SubnetIds, ","))
 	}
-	if spec.EtcdEncryption {
+	if spec.FIPS {
+		command += " --fips"
+	} else if spec.EtcdEncryption {
```

Comment:
I don't understand this because if fips is enabled etcdEncryption should be enabled is what i infer from the above code

Reply:
Yes, but I want it to be implicit - if you set `--fips` you don't need to set etcd encryption, as it will happen automatically, so no need to have both flags.